src/ApiGenerator/RestSpecification/XPack/sql.clear_cursor.json

```
@@ -1,8 +1,7 @@
 {
   "sql.clear_cursor":{
     "documentation":{
-      "url":"https://www.elastic.co/guide/en/elasticsearch/reference/current/sql-pagination.html",
-      "description":"Clears the SQL cursor"
+      "url":"Clear SQL cursor"
```

Bad merge on the ES side?

---

```suggestion
      "url":"https://www.elastic.co/guide/en/elasticsearch/reference/current/sql-rest-overview.html"
```

This is the way it is in `7.6.0` tag on ES